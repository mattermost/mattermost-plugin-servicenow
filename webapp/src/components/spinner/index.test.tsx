// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import Spinner from 'src/components/spinner';

describe('Spinner', () => {
    const baseProps = {
        extraClass: 'mockClassName',
    };

    it('should match snapshot with correct className', () => {
        const wrapper = shallow(<Spinner {...baseProps}/>);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('div').hasClass(baseProps.extraClass)).toBeTruthy();
    });
});
