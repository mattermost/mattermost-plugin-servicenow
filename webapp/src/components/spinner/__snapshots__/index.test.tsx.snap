// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match snapshot with correct className 1`] = `
<div
  className="text-align-center servicenow-spinner-wrapper mockClassName"
>
  <svg
    className="spinner"
    viewBox="0 0 80 80"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="40"
      cy="40"
      r="30"
    />
  </svg>
</div>
`;
